Rack/rack_app.rb

```diff
@@ -0,0 +1,53 @@
+#frozen_string_literal: true
+
+require './config/initializers/initializer'
+require './run.rb'
+
+class RackApp
+  def call(env)
+    request = Rack::Request.new(env)
+
+    case request.path.start_with?
+    when '/api/sups'
+      Handlers::Sup.new(request).response
+    when 'api/customers'
+      Handlers::Customer.new(request).response
+    else
+      Rack::Response.new({ error: 'Not Found' }.to_json, 400, HEADERS).finish
```

Comment:
- Status `Not Found` has status code 404
- I think you can use here method `response_404` from `Handlers::Base`, there is the same logic

Reply:
OMG 400...